hera_cal/lstbin.py

```diff
@@ -641,6 +637,12 @@ def lst_bin_files(data_files, input_cals=None, dlst=None, verbose=True, ntimes_p
                     tarr = time_arrs[j][k]
                     larr[larr < larr[0]] += 2 * np.pi
 
+                    # phase wrap larr to get it to fall within 2pi of file_lists
+                    while larr[0] + 2 * np.pi < fmax:
+                        larr += 2 * np.pi
+                    while larr[-1] - 2 * np.pi > fmin:
```

Comment:
it seems odd to shift down an octave here when we shifted up before. is this necessary?